app/config/FrontendAppConfig.scala

```diff
@@ -109,9 +109,11 @@ class FrontendAppConfig @Inject() (configuration: Configuration, phase5Switch: P
       s"$departureFrontendUrl/local-reference-number"
     }
 
-  val phase5DepartureEnabled: Boolean       = phase5Switch.Departures.enabled
-  val phase5ArrivalEnabled: Boolean         = phase5Switch.Arrivals.enabled
-  val maxErrorsForAmendableDeclaration: Int = configuration.get[Int]("maxErrorsForAmendableDeclaration")
+  val phase5DepartureEnabled: Boolean = phase5Switch.Departures.enabled
+  val phase5ArrivalEnabled: Boolean   = phase5Switch.Arrivals.enabled
+
+  val maxErrorsForAmendableDeclaration: Int     = configuration.get[Int]("maxErrorsForAmendableDeclaration")
+  val maxErrorsForCancellationNotification: Int = configuration.get[Int]("maxErrorsForAmendableDeclaration")
```

Comment:
configuration.get[Int]("maxErrorsForCancellationNotification") instead of configuration.get[Int]("maxErrorsForAmendableDeclaration")

Reply:
Good spot!